ejb3/src/main/java/org/jboss/as/ejb3/subsystem/EJB3AsyncResourceDefinition.java

```diff
@@ -28,30 +28,44 @@
 import org.jboss.as.controller.SimpleAttributeDefinition;
 import org.jboss.as.controller.SimpleAttributeDefinitionBuilder;
 import org.jboss.as.controller.SimpleResourceDefinition;
+import org.jboss.as.controller.capability.RuntimeCapability;
 import org.jboss.as.controller.registry.AttributeAccess;
 import org.jboss.as.controller.registry.ManagementResourceRegistration;
+import org.jboss.as.threads.ThreadsServices;
 import org.jboss.dmr.ModelType;
 
+import java.util.concurrent.ExecutorService;
+
 /**
  * A {@link org.jboss.as.controller.ResourceDefinition} for the EJB async service
  * <p/>
  * @author Stuart Douglas
  */
 public class EJB3AsyncResourceDefinition extends SimpleResourceDefinition {
 
+    // this is an unregistered copy of the capability defined and registered in /subsystem=ejb3/thread-pool=*
+    // needed due to the unorthodox way in which the thread pools are defined in ejb3 subsystem
+    protected static final String THREAD_POOL_CAPABILITY_NAME = ThreadsServices.createCapability(EJB3SubsystemModel.BASE_EJB_THREAD_POOL_NAME, ExecutorService.class).getName();
+
+    public static final String ASYNC_SERVICE_CAPABILITY_NAME = "org.wildfly.ejb3.async";
+    public static final RuntimeCapability<Void> ASYNC_SERVICE_CAPABILITY =
+            RuntimeCapability.Builder.of(ASYNC_SERVICE_CAPABILITY_NAME, true).build();
```

Comment:
This one doesn't seem dynamic as there's only 1 instance of this resource with a fixed name.

Reply:
Done.